CONTRIBUTING.md

```diff
@@ -85,7 +85,7 @@ go get -u gopkg.in/matm/v1/gocov-html
 
 ## Package documentation
 
-UberFx uses [md-to-godoc](https://github.com/sectioneight/md-to-godoc) to
+Fx uses [md-to-godoc](https://github.com/sectioneight/md-to-godoc) to
```

Comment:
We dont use this tool anymore, so you can actually nuke this whole "Package documentation" section.